python-test/requirements.txt

```diff
@@ -0,0 +1,90 @@
+argon2-cffi==21.1.0
```

Comment:
is all of this actually used? would be great to limit this to exactly what we need to keep CI pipelines fast

Reply:
I'll try to filter only the extremely necessary